.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+# Description
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context.
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Document change (fix typo or modifying any markdown files, code comments or anything in the examples folder only)
+
+## All Submissions
+
+- [ ] Have you followed the guidelines in our Contributing document?
+- [ ] Have you provided a description?
+- [ ] Are your changes properly formatted?
```

Comment:
Wouldn't this be automatically enforced by our lint tests?

Reply:
Formatted can be a lot of things outside of the linting tool. I can remove it if you want but I don't think 100% relying on lint tests is good practice.

Reply:
It would, but I think (Gabe can agree) it's a good reminder to ask